+siibra/+core/Region.m

```diff
@@ -0,0 +1,102 @@
+classdef Region < handle
+    properties
+        name
+        id
+        parcellation
+        spaceAndRegionUrl
+        spaces
+        parent
+        children
+    end
+    methods
+        function region = Region(name, id, parcellation, dataset_specs)
```

Comment:
Is there a way to add type hints in MATLAB?

Reply:
[Function argument validation](https://www.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html) can be used to ensure/coerce correct types. 

And I just confirmed it does provide tab completion hints as well...here's an example from the Brain Observatory Toolbox: 

![image](https://user-images.githubusercontent.com/23032671/156220725-e08574f0-579f-4ada-827d-9d1ede1a16be.png)

(this makes me realize that I need to think about best way to handle case-insensitivity...since simply including all reasonable casings leads to tab-completion noise)